.travis.yml

```diff
@@ -3,14 +3,17 @@
 
 language: python
 
-env:
-  - TOXENV=py27
-  - TOXENV=py33
-  - TOXENV=py34
+python:
+  - "2.7"
+  - "3.3"
+  - "3.4"
 
-install: pip install tox
+install:
+  - pip install -U setuptools
+  - pip install tox
 
-script: tox
+script:
+-   tox -e $(echo py$TRAVIS_PYTHON_VERSION | tr -d .)
 
```

Comment:
The changes to this file are not related to switching to `setup.cfg`, so they should be split out into a separate PR (with an explanation of why the changes are needed - are the tests failing as things stand?).